CONTRIBUTING.md

```diff
@@ -403,6 +403,12 @@ system for changes.
 1. Our mage target should open <http://localhost:1313/docs> to preview the
 site/docs.
 
+or use only Hugo
+
+1. Download and install Hugo 0.78.1 extended version. 
```

Comment:
How about we include a link for where they can find Hugo since not all contributors will be familiar with that project

https://github.com/gohugoio/hugo/releases/tag/v0.78.1

Reply:
Done